src/format.py

```diff
@@ -108,6 +110,27 @@ def getScreenIndex(self):
     def getPath(self):
         return self.path
 
+    def getSizeInBytes(self):
+        size = os.stat(self.path).st_size
+        return 'size: ' + str(size) + ' bytes'
+
+    def getLengthInLines(self):
+        num_lines = sum(1 for line in open(self.path))
+        return 'length: ' + str(num_lines) + ' lines'
```

Comment:
no noticeable difference for any files I have, not too surprising.

still gonna assume keeping it at a lower level will be more efficient at some scale--implemented wc -l $filepath